src/singledocparser.h

```diff
@@ -55,6 +57,8 @@ class SingleDocParser {
   anchor_t LookupAnchor(const Mark& mark, const std::string& name) const;
 
  private:
+  using DepthGuard = YAML::DepthGuard<2000>;
```

Comment:
Why do you need this using? You can just inline the call in the .cpp file.

Reply:
Reworked